src/lib/ipr.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const Transport = require('./transport')
+const assert = require('assert')
+const cryptoHelper = require('../utils/crypto')
+
+/**
+  * @module IPR
+  */
+
+/**
+  * Create a payment request for use in the IPR transport protocol.
+  *
+  * @param {Object} params Parameters for creating payment request
+  * @param {String} params.destinationAmount Amount that should arrive in the recipient's account. This value is a string representation of an integer, expressed in the lowest indivisible unit supported by the ledger.
+  * @param {String} params.destinationAccount Target account's ILP address
+  * @param {String} params.receiverSecret Secret for generating IPR packets
+  * @param {String} [params.id=uuid.v4()] Unique ID for the request (used to ensure conditions are unique per request)
+  * @param {String} [params.expiresAt=30 seconds from now] Expiry of request
+  * @param {Object} [params.data=null] Additional data to include in the request
+  * @param {Object} [params.headers=null] Additional headers for private details
```

Comment:
What kind of Object is this? A map with the headers as keys? If so it should say that